components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -311,7 +311,7 @@ class RewardsServiceImpl : public RewardsService,
 
   bool ShouldShowOnboarding() const override;
 
-  void SaveOnboardingResult(OnboardingResult result) override;
+  void EnableRewards() override;
```

Comment:
Maybe here or somewhere else we can just add a quick comment to explain the history behind `SaveOnboardingResult`/`EnableRewards`. Otherwise, I feel like I'm going to forget all of this soon...

Reply:
I was hoping that I *could* forget soon. I'll add a note, though.